server/sonar-server/src/test/java/org/sonar/server/measure/persistence/MeasureDaoTest.java

```diff
@@ -37,13 +37,12 @@
 public class MeasureDaoTest extends AbstractDaoTestCase {
 
   DbSession session;
-
-  MeasureDao dao;
+  MeasureDao sut;
```

Comment:
System Under Test, which represents what you're testing. See http://xunitpatterns.com/SUT.html. It's easy then to identify the unit under test. See org.sonar.application.AppLoggingTest org.sonar.batch.index.SourceDataFactoryTest for use outside of my own.